### ISSUE TITLE
Product showing incorrect status after approval in Shopify

### ISSUE THREAD

**Issue by ZiggyAtl** [ocupop/comission-application#148]
@paulstroot  I received 2 emails telling me that 'The Kiss' and 'Cityscape' had been submitted by the museum. So I followed the instructions in the emails. When I went back to the app the status in the Manage Art section still said In Progress, although in the artwork pages it said 'Status:Active' in red and the blue Request Changes box was showing. However it didn't say it was CoMission'ed

![Screen Shot 2022-11-07 at 10 21 30 PM](https://user-images.githubusercontent.com/112654923/200467521-9ab11737-51fa-47fd-b61d-719535417ca1.png)

![Screen Shot 2022-11-07 at 10 21 48 PM](https://user-images.githubusercontent.com/112654923/200467539-5d553840-db07-4f42-b2e6-4f5c56f4ae62.png)

**Comment by ZiggyAtl** [comment]
I tried it with Paul's Test and the status on the Manage Artwork page went to Membership Active, with the date range showing. And the Status on the artwork page went to Active.

I tried it with Venice at Sunset and the Status on the artwork page is Active, but on the Manage Art page it is Awaiting CoMissioning

**Comment by ZiggyAtl** [comment]
@paulstroot unfortunately I can't test this at the moment as I can't get to the end point without the system hanging when I get to Save.

**Comment by ZiggyAtl** [comment]
@paulstroot I got the email saying that Cleveland Museum wanted to submit 'Pauls Test 2' for approval so I went through the various steps. I then went to the Museum App and the artwork page said Status:Active, but the Manage Art page said Awaiting CoMissioning. So I started sending you this response and I've just noticed 5-10 mins later that it now says Membership Active. It looks like one page updates straight away and the other takes a while.

**Comment by ZiggyAtl** [comment]
@paulstroot I think I've figured out what the problem is. The status in the artwork page updates immediately upon it being approved in Shopify. However for the status to update on the Manage Art page the page has to be resubmitted. I tried clicking off that page and back, and that didn't work, but when I refreshed the page the status updated.

**Comment by paulstroot** [comment]
Yes. Currently there needs to be a refresh in order for the status to be updated. The important thing is that the status gets updated in Shopify. The issue it sounds like you are encountering is that the status is not being displayed properly until you refresh your browser, does that sound correct?

This is on my to-do list for tomorrow. Once saved, I'll probably automatically refresh the browser window so that the new status is displayed. I'll make a note here when it's ready for review.

**Comment by paulstroot** [comment]
This should now be working properly. After you save Art in the museum app, the information and status should be reflected properly where it appears in the "Manage Art" and "Dashboard" list.
